### PR TITLE
Announcement addition

### PR DESCRIPTION
**Pull request by Adamx120**
Adding
 - Death and Troll potion are not allowed!
Editing
 - MarkByron is the owner
- markbyron is the owner

https://github.com/TotalFreedom/TotalFreedomMod/issues/972

### CODE REVIEW

**Comment by Wild1145** [comment]
Looks good to me, however there is no issue that this PR has been linked to.

**Comment by Adamx120** [comment]
@Wild1145 link done

File: src/config.yml [hunk 1]
```diff
@@ -225,12 +225,13 @@ announcer:
     - 'Famous players, such as Notch, are always fake!  We are an offline/cracked server!'
     - 'You may contact TotalFreedoms Support on Twitter!  pravi.us/nGS'
     - 'You may download TotalFreedomMod here: pravi.us/V8c'
-    - 'MarkByron is TotalFreedoms owner.'
+    - 'markbyron is TotalFreedoms owner.'
```
Comment (JeromSar) [inline src/config.yml]: This needs to be capitalized as is. I won't mention the reason here.

**Comment by Adamx120** [comment]
@JeromSar Mention completed check it out under your comment

**Comment by JeromSar** [comment]
@Adamx120 Thanks for contributing. This PR will be merged soon.

**Comment by mysteriace** [comment]
:+1: :dancer: :accept: Accepted PR! :accept: :dancer: :+1:

**Comment by LegendIsAwesomes** [comment]
@JeromSar this still needs changes, he took away the capitals in the 'markbyron is the owner' announcement, the capitals where intentionally done so admins could easily ban imposters, I think it should go back to the original.

**Comment by Adamx120** [comment]
@LegendIsAwesomes It is re-changed to MarkByron *faceplam*

**Comment by LegendIsAwesomes** [comment]
Oops :P Ignore my stupidity didn't notice that.

**Comment by JeromSar** [comment]
@Adamx120 No facepalms in here please.

**Comment by JeromSar** [comment]
This pull request does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and therefore is closed.

**Pull request closed without merging**